source/security/firmware-update.rst

```diff
@@ -60,17 +76,25 @@ Generating capsule
 
 After gathering required firmware binaries, capsule image can be generated using capsule generation tool. Please refer to :ref:`generate-binaries-for-capsule` for details about generating component binaries for capsule.
 
-  Capsule tool (``GenCapsuleFirmware.py``) creates a capsule image that can be processed by |SPN| in firmware update flow.
+  The capsule tool (``GenCapsuleFirmware.py``) creates a capsule image that can be processed by |SPN| in firmware update flow.
 
-  Capsule tool is capable of incorporating multiple firmware images into single capsule binary. Each firmware is identified and included in the capsule image using a 4 byte string. While using the tool for capsule generation, Unique string identifier and associated binary should be provided as input to the tool.
+  The capsule tool is capable of incorporating multiple firmware images into single capsule binary.
 
-  For components inside the container region <4 byte id for component inside container : 4 byte id container id> along with associated component binary should be provided as input to the tool. This format is required for components inside container region because there can be multiple container inside BIOS region
+  Each firmware is identified and included in the capsule image using a string. These strings are constructed as follows:
+
+  * For non-container components, simply use its 4 character string identifier <4 byte comp id>.
+
+  * For container components, use its 4 character container identifier along with its 4 character component identifier <4 byte id for component inside container : 4 byte id container id>.
+
+  These firmware identifier strings should be followed by the assoicated binary path.

```

Comment:
Limitations of the tool using redundant and non-redundant blocks at time is not supported any more